module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml

```diff
@@ -4,8 +4,8 @@ start_position_offensive: [-0.2, 0]
 start_position_defensive: [1.5, 0]
 goalie: false
 
-ball_search_walk_start_speed: 0.5
-ball_search_walk_stop_speed: 0.1
+ball_search_walk_start_speed: 0.0
+ball_search_walk_stop_speed: 0.0
```

Comment:
Not sure if we should do this, I'm not sure how the system would handle a 0 velocity